src/routes/io/io.py

```diff
@@ -1,6 +1,7 @@
 import io
 import time
 import puremagic
+from pydantic import BaseModel
```

Comment:
Why are you importing this?

Reply:
Sorry, It was imported by mistake, I have resolve the issue